src/Microsoft.Crank.Controller/Configuration.cs

```diff
@@ -53,7 +53,7 @@ public class Configuration
 
     }
 
-    public class Scenario
+    public class Service
```

Comment:
Why this rename?

Reply:
when I read my code I was confused, in the yml we have scenarios, but this class represents a named service